dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-reports-reconciliation-summary.controller/helpers.ts

```diff
@@ -13,7 +13,7 @@ import {
 import { UtilisationReportReconciliationSummary, UtilisationReportReconciliationSummaryItem } from '../../../../types/utilisation-reports';
 import { UtilisationReportRepo } from '../../../../repositories/utilisation-reports-repo';
 import { getAllBanks } from '../../../../repositories/banks-repo';
-import { mapReportToSummaryItem } from '../helper';
+import { mapReportToSummaryItem } from './reconciliation-summary-item-mapper';
```

Comment:
None of the other controllers have nested helpers folders, so I'm not particularly inclined too unless is there a pre-established pattern elsewhere that we move them to a folder if there are more than one?

Reply:
I am about to add this pattern in the `get-utilisation-report-reconciliation-details-by-id.controller` because of the complexity of that helper, hence the suggestion. The reason I'd prefer that pattern to having this multiple file setup is that the concept of a `helpers.ts` file living at the same level as another file which contains something functioning as a helper seems counter intuitive to me